themes/OneNord-color-theme.json

```diff
@@ -1309,6 +1310,41 @@
           "foreground": "#88C0D0"
         }
       },
+      {
+        "name": "[Ruby] Superclass inheritance",
+        "scope": "source.ruby entity.other.inherited-class",
+        "settings": {
+          "foreground": "#EBCB8B"
+        }
+      },
+      {
+        "name": "[Ruby] Special method calls, namespaces and keyword method arguments punctuation",
+        "scope": ["source.ruby keyword.other.special-method", "source.ruby meta.function.method.with-arguments constant.other.symbol.hashkey.parameter.function punctuation.definition.constant", "source.ruby meta.function.method.with-arguments punctuation.separator.other"],
+        "settings": {
+          "foreground": "#81A1C1"
```

Comment:
Special method calls, keyword method arguments and namespaces punctation are now blue